trino-connector/integration-test/trino-test-tools/trino-cascading-env/init/hive/init.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+sed -i '$d' /usr/local/sbin/start.sh
+sed -i '$d' /usr/local/sbin/start.sh
```

Comment:
What are these two lines of code doing?

Reply:
remove command line `tail -f /dev/null` in the `/usr/local/sbin/start.sh`

Reply:
We need to changed it.  if anyone add a blank line at the end of file. the script doest not work.

Reply:
Remove these commands.